force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -270,12 +349,21 @@ public with sharing class FlowControlLoopsCollections {
      * @return The position of the word in the list as a string or -1 if the word was not found.
      */
     public static Integer findWordInList(String wordToFind, List<String> words) {
-        // The variable to store the index of the word
-
+        // Initialize the index variable
+        Integer index = -1; // Set default to -1, indicating the word was not found
+    
         // Loop through the list of words
-            // If the current word is the word to find, exit the loop
-
-        return null; // Replace null with the variable you used to store the result
+        for (String word : words) {
+            // Use .equals() to compare the strings
+            if (word.equals(wordToFind)) {
+                // If the word is found, get its index in the list
+                index = words.indexOf(word);
```

Comment:
If you are going to use this method, you wouldnt need the loop at all. This already returns the index or -1. If you wanted to loop (which indexOf is really the easiest here), you would keep track of the index in a variable or just a regular for loop with an index.